man/fi_av.3.md

```diff
@@ -450,6 +502,59 @@ size of the buffer needed to store the address.  This size may be
 larger than the input len.  If the provided buffer is too small, the
 results will be truncated.  fi_av_straddr returns a pointer to buf.
 
+## fi_av_insert_auth_key
```

Comment:
The return code of this function needs to be addressed. Since the API only supports one auth_key insertion per call, there is no need to return a count upon success. Thus, FI_SUCCESS will be returned on success.